docs/security.md

```diff
@@ -528,7 +566,7 @@ replaced with one of the above namespaces.
   <tr>
     <td><code>${ns}.keyStoreType</code></td>
     <td>JKS</td>
-    <td>The type of the key store.</td>
+    <td>The type of the key store. This setting is not applicable to the `rpc` namespace.</td>
```

Comment:
Do we want to include a namespace column instead of adding it this way ?
To indicate which namespace a key is applicable to ?